src/App.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
```

Comment:
It's extra, bc TypeScript already know about React types


```suggestion
  componentDidMount() {
```